jena-arq/src/main/java/org/apache/jena/sparql/exec/QueryExecDataset.java

```diff
@@ -89,6 +89,7 @@ public class QueryExecDataset implements QueryExec
     private long                     timeout2         = TIMEOUT_UNSET;
     private final AlarmClock         alarmClock       = AlarmClock.get();
     private long                     queryStartTime   = -1; // Unset
+    private AtomicBoolean            cancelSignal     = new AtomicBoolean(false);
```

Comment:
Because this has moved to the class, it does not need to be passed as an argument to the constructors of the inner classes.  

Because the classes have members of the same name, I'm getting a compiler warning.

Reply:
Removed the superfluous ctor argument from the inner classes.